HHS/OPRE-OPS
# OPS-677 Submitter's Notes using back end

## What changed

The fake (lorem ipsum) Submitter's Notes have been replaced with real note's from the package

## Issue

#1890 

## How to test

Add notes when submitting BLIs for approval, then go to the Approve Agreement page for that workflow and see the notes

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/127965976/6fff44a9-1743-44b3-9ced-9f6d0fa8f995)

## Definition of Done Checklist


## Links

_If relevant, e.g. for a link to a piece of markdown documentation_
